app/channels/pairer/application_cable.rb

```diff
@@ -0,0 +1,14 @@
+module Pairer
+  module ApplicationCable
+    class Channel < ActionCable::Channel::Base
+    end
+
+    class Connection < ActionCable::Connection::Base
+      #identified_by :current_user
```

Comment:
Currently I dont want to have any identified_by statement. I am assuming this should be ok.